integration_tests/configs/ibc.jsonnet

```diff
@@ -84,6 +84,9 @@ config {
     },
   },
   relayer: {
+    global: {
```

Comment:
need replace to mode when bump up hermes
```
mode: {
  clients: {
    enabled: true,
    refresh: true,
    misbehaviour: true,
  },
  connections: {
    enabled: true,
  },
  channels: {
    enabled: true,
  },
  packets: {
    enabled: true,
    tx_confirmation: true,
  },
}
```

Reply:
I see, is this PR about bump up hermes?

Reply:
no, just to keep old version 1st since miss a `yes` flag when open connection and I don't think it's back portable

Reply:
https://github.com/informalsystems/ibc-rs/pull/2475

Reply:
Is it to fix an existing test or? I still don't get what problem this PR solve.

Reply:
https://rfc.zeromq.org/spec/42/#23-patch-requirements

> A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.

Reply:
I see, so we can live without bump pystarport and hermes for now